sample/Kmd.Logic.Audit.Client.SampleLargeAuditEvents/Program.cs

```diff
@@ -0,0 +1,95 @@
+                .Build()
+                .Get<ProgramConfig>();
+
+            if (string.IsNullOrEmpty(config?.Ingestion?.ConnectionString))
+            {
+                Console.WriteLine("You need to specify a connection string (e.g. --Ingestion:ConnectionString=\"test\")");
+                return;
+            }
+
+            var clientConfig = new Kmd.Logic.Audit.Client.SerilogLargeAuditEvents.SerilogLargeAuditEventClientConfiguration
+            {
+                EventSource = config.Ingestion.EventSource ?? $"{typeof(Program).Assembly.GetName().Name} on {Environment.MachineName}",
+                AuditEventTopic = config.Ingestion.AuditEventTopic,
+                EventHubConnectionString = config.Ingestion.ConnectionString,
+                EnrichFromLogContext = config.Client.EnrichFromLogContext,
+                StorageAccountName = config.Ingestion.BlobAccountName,
+                StorageConnectionString = config.Ingestion.BlobConnectionString,
+                StorageContainerName = config.Ingestion.BlobContainerName
+            };
+
+            var events = new List<AuditEvent>();
+            for (int i = 0; i < 100000; i++)
+            {
+                var data = new AuditEvent();
+                events.Add(data);
+            }
+
+            using (var client = new Kmd.Logic.Audit.Client.SerilogLargeAuditEvents.SerilogLargeAuditEventClient(clientConfig))
+            {
+                var audit = (IAudit)client;
+                var name = typeof(Program).Assembly.GetName().Name;
+                var version = typeof(Program).Assembly.GetName().Version;
+
+                Console.WriteLine(
+                    "Sending {0} ({3} threads) audit events to {1} at {2}",
+                    config.Ingestion.NumberOfEventsToSend,
+                    clientConfig.AuditEventTopic,
+                     clientConfig.EventHubConnectionString,
+                    config.Ingestion.NumberOfThreads);
+
+                var sw = System.Diagnostics.Stopwatch.StartNew();
+                var groupId = Guid.NewGuid();
+
+                if (clientConfig.EnrichFromLogContext == true)
+                {
+                    Console.WriteLine($"The GroupId {groupId} will be attached to each event");
+                }
+
+                using (Serilog.Context.LogContext.PushProperty("GroupId", groupId))
+                {
+                    Enumerable
+                        .Range(0, config.Ingestion.NumberOfEventsToSend)
+                        .AsParallel()
+                        .WithDegreeOfParallelism(config.Ingestion.NumberOfThreads)
+                        .WithExecutionMode(ParallelExecutionMode.ForceParallelism)
+                        .Select(i =>
+                        {
+                            audit.Write("Pre release testing with number {IterationNum} from {Application} v{Version} data {Data}", i, name, version, events);
```

Comment:
Maybe we should change the text "Pre release testing" to something more generic?